build.gradle

```diff
@@ -60,7 +41,9 @@ android {
         debug.setRoot('build-types/debug')
         release.setRoot('build-types/release')
     }
-    lintOptions {
-        abortOnError false
+    defaultConfig {
+        targetSdkVersion 19
+    }
```

Comment:
https://github.com/VTCSecureLLC/ace-android/commit/4d4f2ea27b640fc8042b5f9caf85e6e5b3e725ff
 look like you had it removed here. accidentally?